R/ocean_entry_success.R

```diff
@@ -10,7 +10,7 @@
 #' @source IP-117068
 #' @export
 ocean_entry_success <- function(migrants, month, avg_ocean_transition_month,
-                                .ocean_entry_success_length = fallRunDSM::params$.ocean_entry_success_length,
+                                ocean_entry_success_length = fallRunDSM::params$ocean_entry_success_length,
```

Comment:
these are length based intercepts, similar to small medium large in the survival function. plz keep "."

Reply:
fixed!